src/components/home/SearchBar.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { Button, Input } from 'antd';
+import PropTypes from 'prop-types'; // Import PropTypes
+
+const SearchBar = ({ onSearch }) => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const handleSearch = () => {
+    onSearch(searchQuery);
+  };
+
+  return (
+    <div className="search-bar">
+      <Input
+        placeholder="Search by symbol..."
+        value={searchQuery}
+        onChange={(e) => setSearchQuery(e.target.value)}
+      />
+      <Button onClick={handleSearch}>Search</Button>
+    </div>
+  );
+};
```

Comment:
### These are optional requirements, but I strongly require you to make it. ✍🏻 
- [OPTIONAL] According to the project requirement, the home page should have a list of categories that could be filtered using given parameters, for example, category name. However,  your input search seems not to be working at all. Please Fix this. 👍🏻 

- [ OPTIONAL] According to the project requirements:
The App should have unit tests for several components of the app and some pure functions in Redux code.
>   - The App should have integration tests.
- Kindly make sure to add more unit tests and integration tests using React Testing Librar. ✍🏻 

- [OPTIONAL] You have to improve your design styling to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) including:
* Typography: font face, size, and weight
* Layout: composition and space between elements to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
  - Typography: font face, size, and weight
  - Layout: composition and space between elements especially for the `stockDetail` page. 👇🏻 
 
![image](https://github.com/DevKaranJ/stock-watch/assets/79058364/2e7b75d3-2732-460f-96bb-d7d433be66c5)

- Your Design 👇🏻 

![image](https://github.com/DevKaranJ/stock-watch/assets/79058364/e657900c-c89a-46ad-9a94-6f4974c85a19)